### DISCUSSION TITLE
Barlow Twins 👯‍♂️

### DISCUSSION THREAD

**Discussion by johnyquest7** [KeremTurgutlu/self_supervised#41] [Ideas]
Did you look at the LaCun’s Barlow Twins paper?

**Comment by KeremTurgutlu** [comment]
Yeap it's here waiting to be merged: https://github.com/KeremTurgutlu/self_supervised/pull/40. Feel free to review it!

**Reply by johnyquest7** [reply]
🚀 you are fast. Took few days off next week. Have to try new stuff in your repo.
